src/linemod_train.cpp

```diff
@@ -50,7 +49,6 @@
 #include <object_recognition_core/db/document.h>
 #include <object_recognition_core/db/model_utils.h>
 
-#define USE_GLUT 1
```

Comment:
it's because GLUT worked better for me. But it actually requires an X server. If OSMesa is stable enough and works for you, then let's keep it as is for now, no change. Thx.